feat(em): add troubleshooting index (#4583)
scaleway/docs-content: pages/elastic-metal/troubleshooting/index.mdx

---
meta:
  title: Elastic Metal - Troubleshooting
  description: Find troubleshooting resources for resolving issues with Elastic Metal.
content:
  paragraph: Find troubleshooting resources for resolving issues with Elastic Metal.
dates:
  posted: 2025-03-10
categories:
  - elastic-metal
---

<ProductHeader
  productName="Elastic Metal troubleshooting"
  productLogo="elasticMetal"

>   description="Learn how to fix common issues with Scaleway Elastic Metal servers. Find solutions to help resolve problems and get started."

Reviewer: ```suggestion
  description="Learn how to fix common issues with Scaleway Elastic Metal servers."
```

/>

## Featured Pages

<Grid>
    <DefaultCards
      title="Elastic Metal FAQ"

>       description="General info on Elastic Metal servers"

Author: ```suggestion
      description="General info on Elastic Metal servers."
```

      url="/faq/elastic-metal/"
      label="See more"
    />
    <DefaultCard
      title="Elastic Metal server datasheet"

>       description="This datasheet provides an overview of the technical specifications for the different Elastic Metal server offers."

Reviewer: ```suggestion
      description="Get an overview of the technical specifications for the different Elastic Metal servers."
```

      url="/elastic-metal/reference-content/elastic-metal-datasheet/"
      label="See more"
    />
    <DefaultCard
      title="Troubleshooting common Elastic Metal server access issues"
      description="This page provides guidelines how to troubleshoot the most common server access issues."
      url="/elastic-metal/troubleshooting/troubleshoot-access-issues/"
      label="See more"
    />
</Grid>
<Grid>
    <DefaultCard

>       title="Enabling SSH on Elastic Metal servers running virtualization distributions"

Reviewer: ```suggestion
      title="Enabling SSH on Elastic Metal servers"
```

      description="This guide explains how to enable the SSH server on Virtualization distributions"
      url="/elastic-metal/troubleshooting/enable-ssh-virtualization-distribution/"
      label="See more"
    />
    <DefaultCard
      title="Replacing a failed drive in a RAID1 software RAID"

>       description="This guide explains how to replace a failed drive and rebuild the RAID1 array using the mdadm utility."

Reviewer: ```suggestion
      description="Replace a failed drive and rebuild the RAID1 array using the mdadm utility."
```

      url="/elastic-metal/troubleshooting/replace-failed-drive-software-raid/"
      label="See more"
    />
    <DefaultCard
      title="Elastic Metal"

>       description="Our shared responsibility model clarifies the division of duties between Scaleway and our users."

Reviewer: ```suggestion
      description="Understand the division of duties between Scaleway and our users."
```
